<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmotionTabDialog</class>
 <widget class="QWidget" name="EmotionTabDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>299</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>299</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>299</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    font-family:&quot;微软雅黑&quot;;
}


QWidget#widget
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #edeeef, stop: 1.0 #d4d7da);
}

#btnpanel QPushButton
{
	border:1px solid #00000000;
    font-size:14px;
    color:#FFFFFF;
    padding:5px;
    background-color:#3DCE3D;
}

#btnpanel QPushButton:hover
{
    background-color:#38C738;
}
#btnpanel QPushButton:presse
{
    background-color:#2FB82F;
}
#btnpanel QPushButton:checked
{
    background-color:#FFFFFF;
	color:#666666;
}
#btnpanel QPushButton:disabled
{
    background-color:#9DE69D;
}

QPushButton#setting {
    border : 1px solid #00000000;
    image: url(:/Images/mainpanel_setting_normal.png);
}

QPushButton#setting:hover {
    image: url(:/Images/mainpanel_setting_hover.png);
}
QPushButton#setting:pressed {
    image: url(:/Images/mainpanel_setting_pressed.png);
}

QWidget#scrollArea,#btngroup
{
	border:none;
	background-color:#00000000;
}
QToolButton#leftpage,QToolButton#rightpage{  

background: #3DCE3D;
 color: rgb(0, 0, 0);
border-style:flat;  
 
}  
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="contentpanel" native="true">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>152</width>
           <height>269</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>239</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="emotListPanel" native="true">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>87</width>
           <height>87</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#emotcionpanel
{
    background-color:#E2E4E6;
	border:none;
}

QWidget#emotListPanel
{
    background-color:#AA000000;
	border:none;
}

QWidget#scrollArea_2
{
	border:none;
}


</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="emotcionpanel">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>47</width>
               <height>47</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">

QWidget
{
    font-family:&quot;微软雅黑&quot;;
}

QLabel
{
    color:#333333;
}
#emotcionShop QPushButton
{
	border:1px solid #00000000;
    font-size:14px;
    color:#FFFFFF;
    padding:5px;
    background-color:#3DCE3D;
}

#emotcionShop QPushButton:hover
{
    background-color:#38C738;
}
#emotcionShop QPushButton:presse
{
    background-color:#2FB82F;
}
#emotcionShop QPushButton:checked
{
    background-color:#FFFFFF;
	color:#666666;
}
#emotcionShop QPushButton:disabled
{
    background-color:#9DE69D;
}
</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="emotcionShop" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_3"/>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="btnpanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="leftpage">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="btngroup">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>360</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="btngroup_layout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="emotList">
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>下载表情</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rightpage">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::RightArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_setting" native="true">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>30</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>22</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="setting">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
